x-pack/filebeat/input/salesforce/input.go

```diff
@@ -309,6 +318,8 @@ func (s *salesforceInput) RunEventLogFile() error {
 		return fmt.Errorf("error forming query based on cursor: %w", err)
 	}
 
+	s.log.Infof("Query formed: %s", query.Query)
```

Comment:
how about converting this to debug instead?
i don't think user need this.

Reply:
Yeah but this is crucial to see if the cursors and query formation are working fine. Kept this info so that in case there's ever an issue, we can inspect the query for which the issue happened.

Reply:
done!